base/session_local.hpp

```diff
@@ -15,19 +15,23 @@
 #pragma once
 
 #include <functional>
+#include <utility>
 #include <vector>
 
 namespace husky {
 namespace base {
 
+// Priority class, the higher the value, the higher the priority
+enum class Priority { Level1, Level2 };
```

Comment:
Please change the name with `SessionLocalPriority` to avoid some unexpected conflicts.